### PR TITLE
ENH: Add Chris Keefe bio and image to Team

### PR DESCRIPTION
**Pull request by ChrisKeefe**
Added bio and headshot to /_data/team.yml and /assets/team/ChrisKeefe.png respectively.

### CODE REVIEW

**Comment by ChrisKeefe** [comment]
@thermokarst

File: _data/team.yml [hunk 1]
```diff
@@ -51,6 +51,15 @@
       - database design
       - ux/ui
 
+- name: Chris Keefe
+  title: Student Software Engineer
```
Comment (thermokarst) [inline _data/team.yml]: How about "Student Research Software Engineer"?

File: _data/team.yml [hunk 2]
```diff
@@ -51,6 +51,15 @@
...
+  pic: ChrisKeefe.png
+  github: ChrisKeefe
+  interests:
+      - Wildlife Biology
+      - Human-Computer Interaction
```
Comment (thermokarst) [inline _data/team.yml]: 🤖 🕺

**Review by thermokarst — CHANGES_REQUESTED** [review]
Looks great! One minor question, inline, otherwise this is ready to ship! :shipit:

**Comment by ChrisKeefe** [comment]
Changes made, also added LinkedIn in place of CV.

**Review by thermokarst — APPROVED** [review]

**Pull request merged**